examples/mysql-galera/image/cluster.cnf

```diff
@@ -0,0 +1,12 @@
+[mysqld]
```

Comment:
It's included by mysql in /etc/mysql/conf.d and sets up galera replication. I keep it separate from top-level my.cnf as best practices.